### PR TITLE
Make tests more universal

### PR DESCRIPTION
**Pull request by brybacki**
**What this PR does / why we need it**:
Extracted variables so the test names can be configured.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: tests/framework/framework.go [hunk 1]
```diff
@@ -26,9 +26,21 @@ const (
...
-	artifactsDir string
-	maxFails     int
+	BackupNamespace string
+	FailureCount    int
+	artifactsDir    string
+	maxFails        int
+}
+
+func getBackupNamespaceFromEnv() string {
+	backupNamespace := os.Getenv("KVP_BACKUP_NS")
```
Comment (arnongilboa) [inline tests/framework/framework.go:36]: Consider using common consts for "KVP_BACKUP_NS" and "velero"

Comment (brybacki) [reply tests/framework/framework.go]: done

**Comment by brybacki** [comment]
looks like few assertions need to be improved

**Pull request merged**